docs/Tutorials/Advanced/ticketing/drop.md

```diff
@@ -0,0 +1,390 @@
+
+After `initKeypom` is called, the FC Drop can be created by calling `createDrop` and adding an `fcData` parameter. 
+
+:::tip
+Recall that the private keys being generated using `createDrop` are used to store the assets. These keys are then embedded within a link.
+:::
+
+The primary task in creating the Function Call Drop is to define `fcData`. It is an object with the following properties.
+
+```bash
+fcData
+├── methods
+```
+
+- `FCConfig`: [Specific configurations](../../../keypom-sdk/interfaces/FCConfig.md) for the FC Drop.  
+- `methods`: A vector of all the functions to be called for each key use.  
+
+In this example, only the `methods` parameter will be defined for the sake of simplicity.
+
+`methods` is a 2D array. The outer array defines the vectors of functions to be called **per key use**. The inner array dictates the functions invoked on **each particular key use**. For more information on the `methods` parameter, please see the [TypeDocs](../../../keypom-sdk/interfaces/Method.md)
+
+Each inner element of `methods` represents a function call and requires the following parameters:  
+
+- `receiverId`: The contract receiving the function call.  
+- `methodName`: The function to be called on the receiver contract.  
+- `args`: A stringified JSON object of all the arguments to be passsed into `methodName`.  
+- `attachedDeposit`: The yoctoNear deposit attached to the function call when the key is used.  
+
+### Tailoring to Ticketing Requirements
+
+To ensure each key has two key uses with a passwort protected first use, the following `config` can be added to `createDrop`.
+
+```js
+// 2 Uses per key
+config: {
+    usesPerKey: 2
+},
+
+// Create base password and ensure only first key use is password protected
+basePassword: "event-password",
+passwordProtectedUses: [1],
+```
+
+:::info
+As disucssed in the [introduction](drop.md#introduction), the first key use should not call anything, and the second should call `nft_mint`. To do this, a value of `null` can be passed in to the first index of the `methods` array
+:::
+
+On top of the required `receiverId`, `methodName`, `args`, and `attachedDeposit` parameters, a few others will be used used in `methods` to better utilize the SDK. These parameters tell Keypom where to inject arguments. 
+
+- `accountIdField`: Specifies what field Keypom should auto-inject the claiming account's `accountId`into when calling the function.  
+- `dropIdField`: Specifies what field Keypom should auto-inject the drop's `dropId` into when calling the function.  
+
+These parameters will be used alongisde the [`createNFTSeries`](../../../keypom-sdk/modules.md#createnftseries) SDK method.  
+
+In this case, the drop will be interfacing with an NFT contract deployed at `nft-v2.keypom.testnet`. This is a contract specifically made to work with the SDK to allow you to seamlessly create NFTs and NFT Series to attach to your drops. 
```

Comment:
We should explain the general interface of the mint function at this point and explain that it's important to know the function params etc.. before creating the drop.